Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -523,10 +525,12 @@ def alerts_to_incidents_and_fetch_start_from(alerts, fetch_start_time):
             'occurred': incident_occurred[0] + 'Z',
             'rawJSON': json.dumps(alert)
         }
-        incidents.append(incident)
+        new_last_incident_fetched = alert['_id']
```

Comment:
Lets try and use `.get()` instead of direct access

Reply:
done